src/constants.js

```diff
@@ -41,6 +41,8 @@ module.exports = {
     'lasers',
     'splatter',
     'rainbow',
+    'ripples',
```

Comment:
Let's leave the ripples out of this constant until we are ready to release it to the public (and we've tested it for seizure concerns).